CK3toEU4/Source/Mappers/ReligionDefinitionMapper/ReligionDefinitionMapping.cpp

```diff
@@ -32,9 +32,11 @@ void mappers::ReligionDefinitionMapping::registerKeys()
 			province.clear();
 	});
 	registerKeyword("country_as_secondary", [this](const std::string& unused, std::istream& theStream) {
-		countrySecondary = commonItems::stringOfItem(theStream).getString();
-		if (countrySecondary.size() > 4)
-			countrySecondary = countrySecondary.substr(3, countrySecondary.size() - 4);
+		const auto blobList = commonItems::blobList(theStream).getBlobs();
+		if (blobList.size() > 0)
```

Comment:
Can be combined with the previous line:
```
if (const auto blobList = commonItems::blobList(theStream).getBlobs(); blobList.size() > 0)
```

Reply:
Also instead of blobList.size() > 0 check for !blobList.empty()